scripts/compose.py

```diff
@@ -1728,7 +1773,7 @@ def _content(self):
                 "ELASTIC_APM_FLUSH_INTERVAL=5",
                 "ELASTIC_APM_TRANSACTION_MAX_SPANS=50",
                 "ELASTIC_APM_SAMPLE_RATE=1",
-                "ELASTICSEARCH_URL=http://elasticsearch:9200",
+                "ELASTICSEARCH_URL={}".format(self.es_urls),
```

Comment:
like the logstash case, needs to be joined or an entry picked

Reply:
https://github.com/elastic/apm-integration-testing/pull/557#discussion_r306676939